packages/flutter/lib/src/painting/decoration_image.dart

```diff
@@ -376,6 +376,7 @@ void paintImage({
   bool flipHorizontally = false,
   bool invertColors = false,
   FilterQuality filterQuality = FilterQuality.low,
+  bool isAntiAlias = false,
```

Comment:
Should we assert this isn't `null`? Here and below.

Reply:
Done.